### ISSUE TITLE
PEP 3105 Make print a function in docs/web

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8414]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8414|
|Type|enhancement|
|Created|2016-06-03 21:21:50Z|
|Branch|https://github.com/twisted/twisted/tree/8414-rodrigc-print|

This ticket was split out of [ticket:5812] which was too big to review.

PEP 3105 ( ​​​https://www.python.org/dev/peps/pep-3105/ ) **print** deprecates has changed from a statement to a function.

In Python 2.6 and Python 2.7, the default is print as a statement

```
print "Hello"
```

In Python 3.x, the above example is an error, since only print as a function is available.

To consistently use **print** as a function
on Python 2.6, Python 2.7, and Python 3.x, this should be done:

```
from __future__ import print_function

print("Hello")
```

<details><summary>Searchable metadata</summary>

```
trac-id__8414 8414
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8414_rodrigc_print 8414-rodrigc-print
branch_author__ 
status__closed closed
resolution__fixed fixed
component__web web
keywords__None None
time__1464988910641563 1464988910641563
changetime__1466014078425315 1466014078425315
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/130

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Thanks for the changes. They look good.

Waiting for test results and will merge.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset e847c314e78e41b4767e8e067ed8ce28f4329649
```
#!CommitTicketReference repository="" revision="e847c314e78e41b4767e8e067ed8ce28f4329649"
Merge 8414-rodrigc-print: Use print function in docs/web.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8414
```
